src/content/topics/home/account-security/sso/index.mdx

```diff
@@ -108,6 +108,15 @@ You can configure your identity provider to send the following attributes when t
   </CalloutDescription>
 </Callout> 
 
+
+## Required attributes
+
+We require the NameID field in your SSO provider to be mapped to email addresses. See below for an example of what your SSO provider should be providing LaunchDarkly:
+
+`<saml:NameID Format="urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress">admin@test.com</saml:NameID>`
+
+This is the only required field for LaunchDarkly.
```

Comment:
This is kind of misleading, there are a lot of fields that are required as part of the SAML protocol. I think you could probably just leave this off. If you want to emphasize it you could add something to the custom attributes section below that all of those attributes are optional.